### DISCUSSION TITLE
请问大佬这个报错应该怎么解决（macos）

### DISCUSSION THREAD

**Discussion by 1321312424124** [mikumifa/biliTickerBuy#355] [提问专区]
![1719996404139](https://github.com/mikumifa/biliTickerBuy/assets/171526978/81ac537a-fe43-4e71-83a6-e6279b5ec69a)

**Comment by mikumifa** [comment]
截图再下面一点

**Comment by lxphhh** [comment]
刚解决
<img width="1007" alt="image" src="https://github.com/mikumifa/biliTickerBuy/assets/47101949/835f6fbd-918d-4ba7-88c9-09829ff7aae2">
jing'x镜像地址加点到16g以上
